### DISCUSSION TITLE
Image available at docker hub?

### DISCUSSION THREAD

**Discussion by Systemedic** [mealie-recipes/mealie#2623] [General]
Currently the new single container setup is only being available on ghcr.io. Will this be available at the docker hub? I run mealie on a synology and I can't add the ghcr registry over there.

**Comment by michael-genson** [comment]
Not sure if there are plans to make the new build on docker hub, but you can use the GitHub docker repository on Synology. Are you using the new Container Manager, or are you on the old Docker app?

https://www.synology.com/en-global/dsm/feature/container-manager

If you're using the new Container Manager, you can use docker compose on there (under the "Project" tab):
![image](https://github.com/mealie-recipes/mealie/assets/71845777/b5dd7e6d-fe64-4c59-876f-1563e406ed8d)
![image](https://github.com/mealie-recipes/mealie/assets/71845777/5da3884a-0413-4ff4-aa2a-50eef5e8040d)

(I'm using nightly in the above screenshot, but you can also use the RC image)


If you're not using the new Container Manager, you can still use SSH:
https://kb.synology.com/en-my/DSM/tutorial/How_to_login_to_DSM_with_root_permission_via_SSH_Telnet
